src/pages/ProfilesExplorerView/components/SceneAiPanel/infrastructure/useFetchDotProfiles.ts

```diff
@@ -26,7 +26,9 @@ export function useFetchDotProfiles(dataSourceUid: string, fetchParams: FetchPar
       // TODO: pass a signal options to properly abort all in-flight requests
       return Promise.all(
         fetchParams.map(({ query, timeRange }) =>
-          profileApiClient.get({ query, timeRange, format: 'dot', maxNodes: MAX_NODES }).then(cleanupDotResponse)
+          profileApiClient
+            .get({ query, timeRange, format: 'dot', maxNodes: MAX_NODES })
+            .then((response) => cleanupDotResponse(response as string))
```

Comment:
`ProfileApiClient ` now supports the "dot" and the "json" formats. One returns text and the other, josn.